engine/crates/parser-postgresql/src/registry/context/output/builders.rs

```diff
@@ -55,19 +56,20 @@ impl InputTypeBuilder {
 
         Self {
             input_object_type: InputObjectType::new(name.clone(), []),
-            type_mapping: vec![(name, table_id)],
+            type_mapping: vec![(name.clone(), table_id)],
             field_mapping: Vec::new(),
+            unique_constraint_mapping: Vec::new(),
             relation_mapping: Vec::new(),
             nested: Vec::new(),
         }
     }
 
-    pub(crate) fn push_input_column(&mut self, value: MetaInputValue, column_id: TableColumnId) {
-        self.field_mapping.push((value.name.to_string(), column_id));
-        self.push_non_mapped_input_column(value);
+    pub(crate) fn map_unique_constraint(&mut self, field: &str, constraint_id: UniqueConstraintId) {
```

Comment:
why not `push_unique_constraint` like `push_input_column`? both are "mapping".

Reply:
Yeah, kind of. I'm not creating anything with this, I'm just mapping this field to this constraint.

Reply:
up to you, just struck me reading the PR, but I only have a limited view through it.^^